### PR TITLE
Fixed the JS error due to timepicker

### LINKED ISSUE #15687: Javascript error using timepicker
### Describe the bug

A JS error

### To Reproduce

1. Go to a table with a date and edit the row
2. Enter 'Timestamp()' instead of the time
3. Hit enter, repeat if bug does not occur

### Expected behavior

No error

### Server configuration

- phpMyAdmin version: 4.9.x maybe before

https://demo.phpmyadmin.net/QA_4_9/sql.php?db=sakila&table=film_category&pos=0

<details><summary>trace</summary>
<p>

```json
{
    "pma_version": "4.9.3-dev",
    "browser_name": "CHROME",
    "browser_version": "79.0.3945.79",
    "user_os": "Linux",
    "server_software": "nginx/1.10.3",
    "user_agent_string": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.79 Safari/537.36",
    "locale": "fr",
    "configuration_storage": "disabled",
    "php_version": "7.4.0",
    "script_name": "sql.php",
    "exception_type": "js",
    "exception": {
        "mode": "stack",
        "name": "TypeError",
        "message": "Cannot read property 'settings' of undefined",
        "stack": [
            {
                "line": "292",
                "func": "",
                "context": [
                    "            if (lastException === ex) {",
                    "                processLastException();",
                    "            }",
                    "        }, (stack.incomplete ? 2000 : 0));",
                    "",
                    "        throw ex; // re-throw to propagate to the top level (and cause wind//...",
                    "    }",
                    "",
                    "    report.subscribe = subscribe;",
                    "    report.unsubscribe = unsubscribe;",
                    "    return report;"
                ],
                "column": "",
                "uri": "js/vendor/tracekit.js?v=4.9.3-dev",
                "scriptname": "js/vendor/tracekit.js"
            },
            {
                "func": "s._get",
                "line": "10",
                "column": "25946",
                "context": [
                    "",
                    "(function(t){\"function\"==typeof define&&define.amd?define([\"jquery\"],t):t(j//...",
                    "},_convertPositionTo:function(t,e){e||(e=this.position);var i=\"absolute\"===//...",
                    "t(n.alsoResize).each(function(){var e=t(this),s=t(this).data(\"ui-resizable-//...",
                    "},_destroyIcons:function(){this._removeClass(this.headers,\"ui-accordion-ico//...",
                    "},_updateLabel:function(){var t=this.label.contents().not(this.element[0]);//...",
                    "break;case U[1]-1:T+=\" ui-datepicker-group-last\",I=\" ui-corner-\"+(Y?\"left\"://...",
                    "},_setupEvents:function(){this._off(this.handles),this._on(this.handles,thi//...",
                    "}),o.closing=!0,this._trigger(\"close\",e,{tooltip:i}),o.hiding||(o.closing=!//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery-ui.min.js?v=4.9.3-dev",
                "scriptname": "js/vendor/jquery/jquery-ui.min.js"
            },
            {
                "func": "HTMLInputElement.$.datepicker._doKeyUp",
                "line": "1548",
                "column": "27",
                "context": [
                    "\t* Override key up event to sync manual input changes.",
                    "\t*/",
                    "\t$.datepicker._base_doKeyUp = $.datepicker._doKeyUp;",
                    "\t$.datepicker._doKeyUp = function (event) {",
                    "\t\tvar inst = $.datepicker._getInst(event.target),",
                    "\t\t\ttp_inst = $.datepicker._get(inst, 'timepicker');",
                    "",
                    "\t\tif (tp_inst) {",
                    "\t\t\tif (tp_inst._defaults.timeOnly && (inst.input.val() !== inst.lastVal)) {",
                    "\t\t\t\ttry {",
                    "\t\t\t\t\t$.datepicker._updateDatepicker(inst);"
                ],
                "uri": "js/vendor/jquery/jquery-ui-timepicker-addon.js?v=4.9.3-dev",
                "scriptname": "js/vendor/jquery/jquery-ui-timepicker-addon.js"
            },
            {
                "func": "HTMLInputElement.new_func",
                "line": "265",
                "column": "33",
                "context": [
                    "     */",
                    "    wrap_function: function (func) {",
                    "        if (!func.wrapped) {",
                    "            var new_func = function () {",
                    "                try {",
                    "                    return func.apply(this, arguments);",
                    "                } catch (x) {",
                    "                    TraceKit.report(x);",
                    "                }",
                    "            };",
                    "            new_func.wrapped = true;"
                ],
                "uri": "js/error_report.js?v=4.9.3-dev",
                "scriptname": "js/error_report.js"
            },
            {
                "func": "HTMLInputElement.dispatch",
                "line": "2",
                "column": "42571",
                "context": [
                    "/*! jQuery v3.4.1 | (c) JS Foundation and other contributors | jquery.org/l//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=4.9.3-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "HTMLInputElement.$event.dispatch",
                "line": "374",
                "column": "19",
                "context": [
                    "$event.dispatch = function( event ){",
                    "\tif ( $.data( this, \"suppress.\"+ event.type ) - new Date().getTime() > 0 ){",
                    "\t\t$.removeData( this, \"suppress.\"+ event.type );",
                    "\t\treturn;",
                    "\t}",
                    "\treturn $dispatch.apply( this, arguments );",
                    "};",
                    "",
                    "// event fix hooks for touch events...",
                    "var touchHooks = ",
                    "$event.fixHooks.touchstart = "
                ],
                "uri": "js/vendor/jquery/jquery.event.drag-2.2.js?v=4.9.3-dev",
                "scriptname": "js/vendor/jquery/jquery.event.drag-2.2.js"
            },
            {
                "func": "HTMLInputElement.v.handle",
                "line": "2",
                "column": "40572",
                "context": [
                    "/*! jQuery v3.4.1 | (c) JS Foundation and other contributors | jquery.org/l//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=4.9.3-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            }
        ],
        "incomplete": "false",
        "partial": "true",
        "uri": "sql.php?pos=0"
    }
}
```

</p>
</details>

video-encode

### PR DESCRIPTION
**Pull request by kartik1000**
Signed-off-by: Kartik Kathuria <kathuriakartik0@gmail.com>

### Description

This will fix the JS error which was caused due to timepicker as it was missing the $(document).ready() part which is neccessary while using timepicker and was mentioned by @williamdes .

Fixes #15687 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

File: js/makegrid.js [hunk 1]
```diff
@@ -736,12 +736,14 @@ var makeGrid = function (t, enableResize, enableReorder, enableVisib, enableGrid
             // destroy datepicker in edit area, if exist
             var $dp = $(g.cEdit).find('.hasDatepicker');
             if ($dp.length > 0) {
+                $(document).ready(function () {
```
Comment (williamdes) [inline js/makegrid.js]: will this hook unregister if you change pages using tabs ?

Comment (kartik1000) [reply js/makegrid.js]: I am not getting what the term hook means here. But I have checked and it just works the same even after you have switched pages using tabs. If that is not what you mean can you explain more clearly?

Comment (kartik1000) [reply js/makegrid.js]: @williamdes  anychange to make?

Comment (williamdes) [reply js/makegrid.js]: Hey
I am not sure, maybe it it okay
If you look a the code you will often see unregister events handlers in js files

Comment (kartik1000) [reply js/makegrid.js]: Then maybe we can ask somebody else for approval. Whatever you feel is best :)

Comment (williamdes) [reply js/makegrid.js]: Good idea, I will add the fix to next milestone to not forget it

**Comment by williamdes** [comment]
I had a look another time and this fix seems not to be the best one even if it fixes the issue :)
Still thank you for your contribution @kartik1000 !

**Pull request closed without merging**